.env.example

```
@@ -0,0 +1,24 @@
+# The schema is appended automatically at runtime/build time; do not hardcode it.
+DATABASE_URL="postgresql://user:password@host:5432/database?sslmode=require"
+
+# Optional explicit schema override (CI / migration scripts). When unset the
+# schema is derived from the environment:
+#   VERCEL_ENV=production -> public
+#   VERCEL_ENV=preview    -> preview_<sanitized branch>
+#   otherwise             -> public
+# DATABASE_SCHEMA=
+
+# --- Clerk (authentication) ---------------------------------------------
+NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY="pk_test_..."
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)